airflow/providers/tabular/provider.yaml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-tabular
+name: Tabular
+description: |
+    `Tabular <https://tabular.io/>`__
+
+versions:
+  - 0.0.1
+
+additional-dependencies:
+  - apache-airflow>=2.2.0
+
+integrations:
+  - integration-name: Tabular
+    external-doc-url: https://tabular.io/docs/
+    logo: /integration-logos/tabular/tabular.jpeg
+    tags: [software]
+
+hooks:
+  - integration-name: Tabular
+    python-modules:
+      - airflow.providers.tabular.hooks.tabular
+
+hook-class-names:  # deprecated - to be removed after providers add dependency on Airflow 2.2.0+
+  - airflow.providers.tabular.hooks.tabular.TabularHook
```

Comment:
Is this needed if the provider is already compatible only with >=2.2 ?

Reply:
Great catch!